long-ride-alerts/src/main/java/org/apache/flink/training/exercises/longrides/LongRidesExercise.java

```diff
@@ -98,17 +103,53 @@ public static void main(String[] args) throws Exception {
     @VisibleForTesting
     public static class AlertFunction extends KeyedProcessFunction<Long, TaxiRide, Long> {
 
+        private ValueState<TaxiRide> rideValueState;
```

Comment:
it has to be `transient`, please modify it to avoid Serialization/Deserialization issues

Reply:
done